src/llmcompressor/transformers/finetune/data/base.py

```diff
@@ -105,7 +105,8 @@ def __call__(self, add_labels: bool = True) -> DatasetType:
         dataset = self.rename_columns(dataset)
         logger.debug(f"Dataset after column renaming: {get_columns(dataset)}")
 
-        if "input_ids" not in get_columns(dataset):
+        model_input_names = getattr(self.processor, "model_input_names", ["input_ids"])
```

Comment:
can you add a quick comment here for the condition change

Reply:
Yep